api/test/integration/tavern_utils.py

```diff
@@ -231,6 +231,10 @@ def test_response_is_different(response, response_value, unexpected_value):
     assert response_value != unexpected_value, f"{response_value} and {unexpected_value} shouldn't be the same"
 
 
+def test_save_token_raw_format(response):
+    return Box({'login_token': response.text})
+
+
```

Comment:
Is this function necessary? Can't it be done with the tavern default functions?

Reply:
Replied [here](https://github.com/wazuh/wazuh/pull/10397#discussion_r766791207).